samples/cf-multi-tenant-application/multi-tenant-app/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "multi-tenant-app",
+  "version": "1.0.0",
+  "description": "Example application using the SAP Cloud SDK for JavaScript.",
+  "scripts": {
+    "start": "node -r ts-node/register --inspect src/server.ts",
+    "debug": "node -r ts-node/register --inspect-brk src/server.ts"
+  },
+  "dependencies": {
+    "express": "^4",
+    "@sap/xsenv": "^3.3.2",
+    "cfenv": "^1.2.4",
+    "@sap-cloud-sdk/util": "canary",
+    "@sap-cloud-sdk/connectivity": "canary",
+    "@sap-cloud-sdk/http-client": "canary",
```

Comment:
[req] I would use `^2.x.y`  versions because:
- users might use it as a scaffolder 
- other samples do not use canary

Reply:
oh yes, I can't remember why I used canary there..